scl/Makefile.am

```diff
@@ -15,7 +15,8 @@ SCL_SUBDIRS	= \
 	cisco		\
 	loggly		\
 	logmatic	\
-	snmptrap
+	snmptrap	\
+	windowspatternparser
 SCL_CONFIGS	= scl.conf syslog-ng.conf
```

Comment:
Please call it `windowseventlogdbparser`. We want to keep the short name for the "official" objects, which will be coming soon. (For more information please turn to Varnyu.)